cartesian_impedance_msgs/msg/CartesianDamping.msg

```diff
@@ -0,0 +1,11 @@
+# Damping parameters for the Cartesian Impedance
+# the values are divided into translational and rotational
+# each of the translational and rotational components is defined as Vector3
+# for more info on Vector3 messages please see the following link:
+# http://docs.ros.org/api/geometry_msgs/html/msg/Vector3.html
+
+
+# [N/m]
```

Comment:
Actually on the iiwa and LWR4+ these values are without unit ( Lehr’s damping ratio) and they are set as doubles. I will add this in the explanation. Do you think is better to define them as unitless or most of the robots use N/ (m/s)?